### ISSUE TITLE
can you explain the loss funtion? and how to use it with data?

### ISSUE THREAD

**Issue by alexliyang** [Tencent/tencent-ml-images#31]
I can not understand the loss function , can you explain it?
and if I want to reference, what should I do to prepare the train data?

thanks a lot

**Comment by wubaoyuan** [comment]
@alexliyang  Please wait for our arxiv manuscript, which will provide a detailed definition of the loss function.

**Comment by alexliyang** [comment]
thx a lot

Now I want to train a classical model,  samples do not use bbox , only classic labels, how to use your loss function?

Thanks

**Comment by wubaoyuan** [comment]
@alexliyang  In the provided code, we also didn't use bbox. So you can directly copy the code of the loss function. Our Arxiv manuscript is expected to be released in one week.

**Comment by alexliyang** [comment]
OK, thanks, I will use this loss to train my model.

and another question: my class_num is 10, whether I should add 1 to class_num i.e. class_num = 11 when I train the model or not ?

Thanks

**Comment by wubaoyuan** [comment]
@alexliyang You mean adding a rejection class? We have not tried that. The main difficulty may be how to obtain the training sample for the rejection class. If you train with 10 classes, if the posteriors of all classes are lower than a threshold, it can be considered as rejection.

**Comment by alexliyang** [comment]
@wubaoyuan  thanks, although I do not understand your loss function , now I copy it and use it to train my model, I find that ,the every step loss value and mean loss value are  smaller than the value from softmax_cross_entropy_with_logits function, but the test acc is not better than using softmax_cross_entropy_with_logits function.
Maybe I used it not correctly

**Comment by wubaoyuan** [comment]
@Jason-xin Already finished, in proof-reading. Maybe one week.

**Comment by WillLiGitHub** [comment]
you can see the function: torch.nn.BCEWithLogitsLoss()
